src/config/mod.rs

```diff
@@ -16,6 +16,7 @@ use crate::modules::script::ScriptModule;
 use crate::modules::sysinfo::SysInfoModule;
 #[cfg(feature = "tray")]
 use crate::modules::tray::TrayModule;
+use crate::modules::upower::UpowerModule;
```

Comment:
This should have a cfg attr:

```suggestion
#[cfg(feature = "upower")]
use crate::modules::upower::UpowerModule;
```